OrbitGl/ApplicationOptions.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORIBIT_GL_APPLICATION_OPTIONS_H_
+#define ORIBIT_GL_APPLICATION_OPTIONS_H_
+
+#include <string>
+
+// The structure used to store Oribit Client Application options
+// The default values are set by main() and passed over to App
+// calss initialization method.
+struct ApplicationOptions {
+  // The host and port of the collection service
+  std::string asio_server_address;
```

Comment:
Why is "asio" part of the name here?

Reply:
for some time we will have both - grpc and asio, once migration is complete I will drop this one.